packages/fast-css-editor-react/src/position/position.style.ts

```diff
@@ -0,0 +1,91 @@
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { toPx } from "@microsoft/fast-jss-utilities";
+
+export interface ICSSPositionClassNameContract {
+    cssPosition: string;
+    cssPosition_selectContainer: string;
+    cssPosition_selectContainer_select: string;
+    cssPosition_input: string;
+    absoluteInput_row: string;
+    absoluteInput_row_center: string;
+    absoluteInput_row_center__activeTop: string;
+    absoluteInput_row_center__activeBottom: string;
+    absoluteInput_row_center__activeLeft: string;
+    absoluteInput_row_center__activeRight: string;
+}
+
+const styles: ComponentStyles<ICSSPositionClassNameContract, {}> = {
+    cssPosition: {
+    },
+    cssPosition_selectContainer: {
+        display: "flex",
+        position: "relative",
+        "&::before, &::after": {
+            top: toPx(12),
+            width: toPx(1),
+            height: toPx(10),
+            content: "''",
```

Comment:
I think you may have to do it like" `content: "\"\""`

Reply:
No need, this is how we are currently doing it.

Reply:
It's actually mixed. @nicholasrice changed some JSS to this, so maybe he can shed some light on it